js/binary/utils.js

```diff
@@ -971,10 +971,6 @@ jspb.utils.byteSourceToUint8Array = function(data) {
     return /** @type {!Uint8Array} */(new Uint8Array(data));
   }
 
-  if (data.constructor === Buffer) {
```

Comment:
I think we should go with the logic on master that looks like this:

```
  if (typeof Buffer != 'undefined' && data.constructor === Buffer) {
    return /** @type {!Uint8Array} */ (
        new Uint8Array(/** @type {?} */ (data)));
  }
```

That should do the right thing whether `Buffer` exists or not.

Reply:
Done.